server/container/guice/jpa-smtp-mariadb/src/test/java/org/apache/james/JPAJamesServerTest.java

```diff
@@ -74,11 +67,7 @@ public void teardown() {
 
         return GuiceJamesServer.forConfiguration(configuration)
                 .combineWith(JPAJamesServerMain.JPA_SERVER_MODULE, JPAJamesServerMain.PROTOCOLS)
-                .overrideWith(
-                        new TestJPAConfigurationModule(mariaDBUrl),
-                        (binder) -> binder.bind(EntityManagerFactory.class)
```

Comment:
this binding statement binds `EntityManagerFactory` to an `EntityManagerFactory` pointing to embedded Derby which I think is not appropriate when testing with mariaDB. So I removed it